packages/components/pie-button/test/helpers/page-object/pie-button-form-integration.page.ts

```diff
@@ -0,0 +1,188 @@
+import { type Locator, type Page } from '@playwright/test';
+import { BasePage } from '@justeattakeaway/pie-webc-testing/src/helpers/page-object/base-page.ts';
+import { form } from './pie-button-form-integration-selectors.ts';
+
+// Define the FormInput type with specific options for userPaymentCardType
+export type FormInput = {
```

Comment:
Does this need to be exported?

Reply:
Yep, this is being used in the test spec.